### PR TITLE
Switch UserCreditPathway to use pathway foreign key

### PR DESCRIPTION
**Pull request by bbaker6225**
Note: There is some renaming of credit pathway to just pathway, but the majority of that will come when we completely get rid of CreditPathway.  This PR is just making sure that UserCreditPathway's credit_pathway foreign key is not used anywhere, in preparation for deleting it in favor of pathway.

Credentials Pull Request
---

Make sure that the following steps are done before rebasing/merging

  - [ ] Request a review if desired
  - [ ] Squash/Fixup your branch to one commit
  - Config/Dependency Changes (e.g. config files, scripts that are used for provisioning etc.)
    - [ ] Make sure you have updated [edx/configuration](https://github.com/edx/configuration) and [edx/devstack](https://github.com/edx/devstack) if necessary
    - [ ] Make sure the change builds successfully in a sandbox
  - UI Changes 
    - [ ] Consider other browsers (e.g. Firefox)
    - [ ] Check mobile view
    - [ ] Consider accessibility (e.g. Run aXe on the page)

### CODE REVIEW

File: credentials/apps/records/management/commands/seed-records.py [hunk 1]
```diff
@@ -313,7 +313,7 @@ def seed_user_credit_pathways(user, credit_pathways):
         has sent an email to that pathway """
         user_credit_pathway, created = UserCreditPathway.objects.get_or_create(
             user=user,
-            credit_pathway=credit_pathways[1],
+            pathway=credit_pathways[1],
```
Comment (mikix) [inline credentials/apps/records/management/commands/seed-records.py]: That's assigning a CreditPathway object to a Pathway foreign key right?

Comment (bbaker6225) [reply credentials/apps/records/management/commands/seed-records.py]: Good catch, I'll fix this

Comment (bbaker6225) [reply credentials/apps/records/management/commands/seed-records.py]: Should be good now

File: credentials/apps/records/management/commands/seed-records.py [hunk 2]
```diff
@@ -289,31 +289,33 @@ def seed_program_cert_records(user, programs, faker):
...
-        """ Seed two credit pathways """
-        all_program_pathway, created = CreditPathway.objects.get_or_create(
+    def seed_pathways(site, programs, faker):
+        """ Seed two pathways """
+        all_program_pathway, created = Pathway.objects.get_or_create(
             site=site,
             name='All program pathway',
-            org_name="MIT")
+            org_name="MIT",
+            uuid=faker.uuid4())
```
Comment (mikix) [inline credentials/apps/records/management/commands/seed-records.py]: Ah thank you. I had the same fix in my acceptance branch.

**Pull request merged**